### DISCUSSION TITLE
Naked Gopro 11 Wont reconize battery

### DISCUSSION THREAD

**Discussion by RichardMattBal** [gopro/labs#999] [Q&A] [answered]
Hi i have a odd issue with a gopro hero 11. It used to work fine , bought it second hand to make it naked as i fly drones commercially. Up to yesterday it booted with a battery normally. Today when i decided to make it naked i wanted to install gopro labs first. Suddenly no reaction from the gopro when a normal battery is used. Only with usb it powers on normally. Strange. I decided to open it up hoping i could solve this issue and install labs later with the battery cage connected. Now its fully naked, fully functional on usb, connected to my app which also wants me to update it, and i cant install labs as no matter what i do it wont turn on using the battery cage and battery. Only via usb which gives me an update error. I can measure 3.87v on the battery cage flat cable connector so that parts working. It must be a faulty regulator or blown diode i suspect. Now finally my question, is there any any way to install labs without a original battery? For now this (still working) camera is rendered useless if i cant install labs. I hope someone can help

**Comment by dnewman-gpsw** [comment, marked as answer]
Unfortunately non-Labs FW requires the battery authentication, and as that the battery/cage is not working in your setup, I can't see how to work around that.  Can you try someone's else spare cage and battery, just in case that is the issue?

**Reply by RichardMattBal** [reply]
That's very unfortunate to hear. Unfortunately ive already tried with multiple batteries.  It seems this board has a issue on the battery power side which doesnt show up on the usb power side. The only qay i can provide it power is through the usb c side of the board. Ill try another cage but to be honest i doubt that that will be the issue. Thanks anyways for your time and reply
